docs/run/primitives-get-started.mdx

```diff
@@ -126,24 +199,79 @@ Again, we use the transpiler to get an ISA circuit.
 pm = generate_preset_pass_manager(backend=backend, optimization_level=1)
 isa_circuit = pm.run(circuit)
 ```
+  </TabItem>
 
-### 3. Initialize the Qiskit Runtime Sampler
+  <TabItem value="SamplerV1" label="Sampler (V1)">
+    ```python
+import numpy as np
+from qiskit.circuit.library import IQP
+from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+from qiskit.quantum_info import random_hermitian
 
-Here we are initializing an instance of `qiskit_ibm_runtime.Sampler` instead of `qiskit.primitives.Sampler` to use Qiskit Runtime's implementation of the `Sampler`. 
+n_qubits = 127
 
-When you initialize the `Sampler`, you'll need to pass in the simulator or system you previously selected as the target device (or simulator).  You could also do this within the `session` parameter. 
+mat = np.real(random_hermitian(n_qubits, seed=1234))
+circuit = IQP(mat)
+circuit.measure_all()
+```
+Again, we use the transpiler to get an ISA circuit.
 
 ```python
+pm = generate_preset_pass_manager(backend=backend, optimization_level=1)
+isa_circuit = pm.run(circuit)
+```
+  </TabItem>
+</Tabs>
+
+### 3. Initialize the Qiskit Runtime Sampler
+
+Here we are initializing an instance of `qiskit_ibm_runtime.Sampler` instead of `qiskit.primitives.Sampler` to use the Qiskit Runtime `Sampler`. 
```

Comment:
```suggestion
Here we initialize an instance of `qiskit_ibm_runtime.Sampler` instead of `qiskit.primitives.Sampler` to use the Qiskit Runtime `Sampler`. 
```